.github/workflows/main.yml

```diff
@@ -0,0 +1,19 @@
+name: Check PR title and project linting
+on:
+  pull_request:
+    types:
+      - opened
+      - reopened
+      - edited
+      - synchronize
+
+jobs:
+  lint:
+    if: "!contains(github.event.pull_request.head.ref, 'renovate') && !contains(github.event.pull_request.head.ref, 'dependabot')"
+    runs-on: ubuntu-latest
+    steps:
+    - uses: deepakputhraya/action-pr-title@master
+      with:
+        regex: '^(CIV-[0-9]{3,4} [a-zA-Z0-9._\- ]+)|(\[Snyk\].+)$' # Regex the title should match.
```

Comment:
In the past couple of weeks at some point I had to work on an activity that didn't have a ticket associated with it. I had to put CIV-0000 to let it go through.
In a separate ticket to start enforcing these conventions also on our commits, during discussion an exception to the rule was brought up to allow CIV-TASK for those activities that are not worth creating a ticket for.
This is the ticket my devs will be using when trying to enforce the conventions on our git commit messages and branch names:
https://tools.hmcts.net/jira/browse/CIV-6995

Not required at the moment that you modify your conventions, but do take a look at CIV-6995 and see if there's any convention you would want to align now.